registry/application/services/organization_publisher_service.py

```diff
@@ -219,4 +225,16 @@ def update_verification(self, verification_type, verification_details):
         else:
             logger.error(f"Invalid verification type {verification_type}")
             raise MethodNotImplemented()
+
+        # TODO send_email should not have boto_utils
+        try:
+            if organization:
+                contacts = []
+                for contact in organization.contacts:
+                    contacts.append(contact['email_id'])
+                send_email_notification(contacts, ORG_APPROVE_SUBJECT.format(organization.name), ORG_APPROVE_MESSAGE.format(organization.name), NOTIFICATION_ARN,
+                                        self.boto_utils)
+        except:
```

Comment:
Lets also add the contacts to this error so we can identify what went wrong.

Reply:
done